hub/management/commands/import_council_rural_classification.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import add_gss_codes, filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+shades = [
+    {"title": "Sparse and rural", "shader": ""},
+    {"title": "Rural", "shader": ""},
+    {"title": "Urban", "shader": ""},
+    {"title": "Urban with rural areas", "shader": ""},
+]
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council rural/urban classification"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Urban Rural Classification",
+        "description": "A composite measure of 'ruralness' (based population density, settlement size, and drive times) standardised across the countries of the UK.",
```

Comment:
Typo here. Should be: `A composite measure of ‘ruralness’ (based on population density, settlement size, and drive times) standardised across the countries of the UK.`

Reply:
it only uses the based population.